testr/runner.py

```diff
@@ -12,7 +12,7 @@ class TestError(Exception):
 
 
 def testr(*args, **kwargs):
-    """
+    r"""
```

Comment:
I forgot to ask, what's the deal with the "r"s?

Reply:
Eh. I suppose you had some display reason for changing these to raw strings.  Didn't really line up with rest of the intent of the PR but not an issues for me and already merged.